taxi/urls.py

```diff
@@ -16,8 +22,25 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/create/",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create"
+    ),
+    path(
+        "manufacturers/<int:pk>/update/",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update"
+    ),
+    path("manufacturers/<int:pk>/delete/",
+         ManufacturerDeleteView.as_view(),
+         name="manufacturer-delete"
```

Comment:
The path for 'manufacturer-delete' should be formatted with each argument on a new line for consistency with other paths. Consider using the format shown in the checklist: path('manufacturers/<int:pk>/delete/', ManufacturerDeleteView.as_view(), name='manufacturer-delete'),